task-8/src/api/services/products.service.ts

```diff
@@ -1,13 +1,13 @@
 import { productRepository } from "../repositories";
 
-export const getAllProducts = () => {
-  const products = productRepository.findAllProducts();
+export const getAllProducts = async () => {
+  const products = await productRepository.findAllProducts();
 
   return products;
 };
 
-export const getProductById = (productId: string) => {
-  const product = productRepository.findProductById(productId);
+export const getProductById = async (productId: string) => {
+  const product = await productRepository.findProductById(productId);
```

Comment:
praise: If not considering the additions of async/await, then this is what changing the data source for the application should look like. No changes should occur at the service level 👍